helix-core/src/main/java/org/apache/helix/api/status/ClusterManagementMode.java

```diff
@@ -67,4 +69,26 @@ public Status getStatus() {
     public Type getMode() {
         return mode;
     }
+
+    public boolean isFullyInNormalMode() {
```

Comment:
Question, do we have a term for partially normal status, or what does that stand for?

I assume it should be,
1. NORMAL, COMPLETE,
2. CLUSTER_PAUSE, IN_PROGRESS
3. CLUSTER_PAUSE, COMPLETE
4. CLUSTER_PAUSE, IN_PROGRESS ---> it is recovering from pause, but since it is still partially paused, I think to mark it CLUSTER_PAUSE is more appropriate.
5. NORMAL, COMPLETE

Reply:
What I'd like to have is:
1. What is the desired/target mode?
2. What is the status, IN PROGRESS or COMPLETED?

`isFullyInNormalMode` means the cluster is stable in the normal mode. For the situation: recovering from pause, it should be IN_PROGRESS to NORMAL. It's already set to NORMAL mode, so we want to mark it NORMAL, IN PROGRESS.